ops/Dockerfile

```diff
@@ -123,16 +123,18 @@ ENV BUILD_TESTS=$BUILD_TESTS
 ARG BUILD_EXAMPLE
 ENV BUILD_EXAMPLE=$BUILD_EXAMPLE
 
-ARG TRITON_REPO_ORGANIZATION=https://github.com/triton-inference-server
+ARG TRITON_REPO_ORGANIZATION=$TRITON_REPO_ORGANIZATION
```

Comment:
This line is not valid, since `TRITON_REPO_ORGANIZATION` is being used here without first being defined elsewhere. Is `TRITON_REPO_ORGANIZATION` somewhere else?

Reply:
Currently, `https://github.com/triton-inference-server` is the default value for the argument `TRITON_REPO_ORGANIZATION`. You can override it by passing `-DTRITON_REPO_ORGANIZATION=...` when invoking CMake.

Reply:
yes, https://github.com/triton-inference-server/fil_backend/blob/06d9fb7ae89aa17be5f7e0867e1537950d01974f/CMakeLists.txt#L23

Reply:
I'm following the format specified here: https://github.com/triton-inference-server/fil_backend/pull/409/files#diff-d2e93d579aa110eb1f1fb1526ea0d7ca3fe7cbb3ed700ad8826aae45341a43beL128. I could change it back to the default case if you like?

Reply:
That line is different, since `TRITON_VERSION` was defined earlier in the Dockerfile. On the other hand, `TRITON_REPO_ORGANIZATION` is not defined earlier in the Dockerfile, so a default value needs to be supplied.

Reply:
The referenced line constructs a new string from a defined variable by adding an `r` in front, whereas the new changes are equivalent to setting the variables to empty strings.

Reply:
sorry for my mistake. Changed